### PR TITLE
Make IP configurable and default to 127.0.0.1

### PR DESCRIPTION
**Pull request by drewkerrigan**
- Added `navstar_rest.ip` to `sys.config`
- Added `{ip, Ip}` to `cowboy:start_http` in `navstar_rest_app`

### CODE REVIEW

**Comment by sargun** [comment]
@drewkerrigan Please rebase

**Comment by drewkerrigan** [comment]
@sargun rebase complete

File: config/sys.config [hunk 1]
```diff
@@ -25,6 +25,7 @@
   },
   {navstar_rest,
     [
+      {ip, {127,0,0,1}},
```
Comment (sargun) [inline config/sys.config]: spaces?

**Comment by sargun** [comment]
@drewkerrigan Do you want to try to do a dcos/dcos build with this branch to verify your changes took effect?

**Comment by drewkerrigan** [comment]
@sargun yes, will do

**Comment by drewkerrigan** [comment]
Test in progress: https://github.com/dcos/dcos/pull/582

**Comment by sargun** [comment]
Squash? And then I'll merge.

**Comment by drewkerrigan** [comment]
@sargun squashed

**Pull request merged**